docs/plugins/inputs/snmp.asciidoc

```diff
@@ -64,6 +64,37 @@ $ smidump --level=1 -k -f python RFC1213-MIB > RFC1213-MIB.dic
 
 Note that the resulting file as output by `smidump` must have the `.dic` extension.
 
+As `smidump` only looks for mib dependencies in its pre-configured path lists as defined in the documentaion in the https://www.ibr.cs.tu-bs.de/projects/libsmi/smi_config.html["MODULES LOCATIONS"] section, you may need to provide the paths to locations of MIBs in your particular environment to avoid a `failed to locate MIB module` error.
+
+Two ways to achieve this are by using an environment variable or a config file to provide the additional path configuration.
+
+Set the `SMIPATH` env var with the path to your mibs ensuring you include a prepended colon `:` for the path, for example:
+
+[source,sh]
+-----
+$ SMIPATH=":/path/to/mibs/" smidump -k -f python CISCO-PROCESS-MIB.mib > CISCO-PROCESS-MIB_my.dic
+-----
+
+NOTE: SMI docs on the prepended colon:
+
+  The ‘path’ command argument and the environment variable either start with a path separator character (‘:’ on UNIX-like systems, ‘;’ on MS-Windows systems) to append to the path, or end with a path separator character to prepend to the path, or otherwise completely replace the path.
```

Comment:
Not formatting as intended. Reached out to contributor to find out intention. PR to change source will be needed.